public/app/features/profile/UserProfileEditForm.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { FC } from 'react';
+import { Trans } from '@lingui/macro';
```

Comment:
**Just an idea:** maybe it would worth considering using something like a "Facade" for these imports

What I mean is to have a core module that we are importing from, and this core module would basically proxy / re-export the third-party functionalities that we are depending on. Like this we could more easily refactor the code if we decide to transition to an other framework or library in the future.

E.g.:
```javascript
import { Trans, I18nProvider } from './core/localisation';
```

Reply:
Great suggestion! I moved `I18nProvider` to this pattern. I think `Trans` must remain as the existing import as the code is statically analyzed to look for it to extract the strings.